CHANGELOG.md

```diff
@@ -1,20 +1,54 @@
 # Changelog
 
-## 0.2.35
-- Changed lambda `Term` with bind patterns to display as `let` terms.
+All notable changes to this project will be documented in this file.
 
-## 0.2.34
-- Added `<=` and `>=` operators. (#451)
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
I don't want to commit at this point to follow semantic versioning.
In the future we might decide to follow a different versioning scheme, like many (most?) programming languages do.